lib/task_helpers/imports/custom_buttons.rb

```diff
@@ -0,0 +1,88 @@
+require "English"
+
+module TaskHelpers
+  class Imports
+    class CustomButtons
+      class ImportArInstances
+        DEBUG_MODE = false
+
+        def self.import(obj_hash)
+          new.import(obj_hash)
+        end
+
+        def import(obj_hash)
+          ActiveRecord::Base.transaction { obj_hash.each { |obj_def| create_object(*obj_def) } }
+        end
+
+        def create_object(class_name, obj_array)
+          klass = class_name.camelize.constantize
+
+          obj_array.collect do |obj|
+            create_unique_values(obj) if DEBUG_MODE
```

Comment:
Drop the `DEBUG_MODE` constant and the `create_unique_values` method.  These were just for testing during development.

Reply:
Thanks for getting rid of the constant.  Now we should also remove the usage of it.  😄